cmd/webhook/main.go

```diff
@@ -126,11 +138,12 @@ func NewConfigValidationController(ctx context.Context, _ configmap.Watcher) *co
 			// metrics.ConfigMapName():   metricsconfig.NewObservabilityConfigFromConfigMap,
 			logging.ConfigMapName():        logging.NewConfigFromConfigMap,
 			leaderelection.ConfigMapName(): configvalidation.ValidateLeaderElectionConfig,
+			authorization.ConfigName:       authorization.NewDefaultsConfigFromConfigMap,
```

Comment:
for consistency, want to have a `ConfigMapName()` accessor instead?

Reply:
Done.